packages/app/src/app/pages/Dashboard/Components/Folder/index.tsx

```diff
@@ -135,7 +135,13 @@ export const Folder = (folderItem: DashboardFolder) => {
   const [newName, setNewName] = React.useState(name);
 
   const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setNewName(event.target.value);
+    const { value } = event.target;
+    if (value && value.trim()) {
+      event.target.setCustomValidity('');
```

Comment:
Do we need to set empty validity?

Reply:
Yes, this is required. Otherwise, this will show the validation message even if the input is correct. Check this video: https://www.loom.com/share/93e2eaf57ee447bdb886eaf5736da4a0

You can check the behavior on this sandbox: https://codesandbox.io/s/agitated-albattani-joqpb?file=/src/App.js

Reply:
oh wow, that's annoying :o